packages/typespec-client-generator-core/src/context.ts

```diff
@@ -135,6 +139,34 @@ export function createTCGCContext(
       }
       return getVersions(this.program, namespaces[0])[1]?.getVersions()?.[0].enumMember.enum;
     },
+    getClients(): SdkClient[] {
```

Comment:
Is it supposed to be called by emitter?

Reply:
emitter could call it. actually, it's kind of early version to tcgc typekit.